resolver-functions/SampleForReport.ts

```diff
@@ -223,68 +229,79 @@ export const SampleForReportResolver = async (root, args, context)=> {
-      taxon => {
+  const nextGenWorkflowRuns: query_SampleForReport_workflow_runs_items[] =
+    workflowsWorkflowRuns.map(workflowRun => {
+      const consensusGenome = consensusGenomes.find(consensusGenome => {
+        return consensusGenome.node.producingRunId === workflowRun.id;
+      });
+      const { sequencingRead } = consensusGenome?.node || {};
+      const parsedRawInputsJson = JSON.parse(workflowRun.rawInputsJson || "{}");
+      // Taxon will only be null if the user selected "unknown" for taxon in WGS upload
+      const taxon = taxonInfo.find(taxon => {
         const workflowTaxId = workflowRun.entityInputs.edges.find(
-          input => input.node.entityType === "taxon"
+          input => input.node.entityType === "taxon",
         )?.node.inputEntityId;
         return taxon.id === workflowTaxId;
-      }
-    );
-    // Accession will be null for WGS uploads
-    const accession = accessionInfo.find(
-      accession => 
-        accession.id === workflowRun.entityInputs.edges.find(
-          input => input.node.entityType === "accession"
-        )?.node?.inputEntityId
-    );
-    // If !consensusGenome this is a workflow run that is in progress
-    return {
-      deprecated: null,
-      executed_at: workflowRun.createdAt,
-      id: workflowRun.id,
-      input_error: workflowRun.errorMessage,
-      inputs: {
-        accession_id: accession?.accessionId,
-        accession_name: accession?.accessionName,
-        creation_source: parsedRawInputsJson?.creation_source,
-        ref_fasta: parseRefFasta(
-          consensusGenome?.node?.referenceGenome?.file?.path,
-        ),
-        taxon_id: taxon?.upstreamDatabaseIdentifier, // this is the NCBI taxid, which should match the taxids used in the mNGS report
-        taxon_name: taxon?.name,
-        technology: sequencingRead?.technology,
-      },
-      rails_workflow_run_id: workflowRun.railsWorkflowRunId?.toString(), // this is added for deduplicating below
-      run_finalized: isRunFinalized(workflowRun.status || ""),
-      status: workflowRun.status,
-      wdl_version: workflowRun.workflowVersion?.version,
-      workflow: workflowRun.workflowVersion?.workflow?.name,
-    };
-  });
+      });
+      // Accession will be null for WGS uploads
+      const accession = accessionInfo.find(
+        accession =>
+          accession.id ===
+          workflowRun.entityInputs.edges.find(
+            input => input.node.entityType === "accession",
+          )?.node?.inputEntityId,
+      );
+      // If !consensusGenome this is a workflow run that is in progress
+      return {
+        deprecated: null,
+        executed_at: workflowRun.createdAt,
+        id: workflowRun.id,
+        input_error:
+          workflowRun.errorLabel != null || workflowRun.errorMessage != null
+            ? {
+                label: workflowRun.errorLabel,
+                message: workflowRun.errorMessage,
+              }
+            : undefined,
```

Comment:
is this the main change in this file?  I think there's a lot of other spacing or minor changes, so wanted to make sure that this is the primary change.

Reply:
Yep, this is the main change

It'd prolly be a good idea for us to add Prettier to the auto eslint format bot if that's possible...